README.md

```diff
@@ -483,14 +541,89 @@ that can be used to submit SQL queries. The endpoint is accessible under
 SQL statements are sent as the body of POST requests in JSON format, where the
 SQL statement is the value of the `stmt` attribute.
 
-### Read Schemas
+**QuantumLeap** offfers an API wrapping CrateDB backend so you can also perform
+multiple types of queries directly talking to QuantumLeap. The documentation of
+the API is [here](https://app.swaggerhub.com/apis/smartsdk/ngsi-tsdb/). Mind the
+versions. If you have access to your quantumleap container (e.g. it is running
+in locahost or port-forwarding to it), you can navigate its API via 
+[http://localhosts:8668/v2/ui](http://localhosts:8668/v2/ui).
+
+When to query **CrateDB** and when **QuatumLeap**?. As a rule of thumb, prefer 
```

Comment:
Typo QuatumLeap

Reply:
Maybe this section should be placed in a sidebar?

This can be done by add it as a Quote

Reply:
Got it